corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml

```diff
@@ -75,14 +75,12 @@
   description: "Upload a file to serve as a CommCare home screen logo on Android phones"
```

Comment:
Yes, the decision was made to not support Java unless a client specifically requests it.